pulp_container/app/migrations/0019_DATA_distribution_model_swap.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 2.2.19 on 2021-03-19 17:42
+
+from django.db import migrations, models, transaction
+import django.db.models.deletion
+
+
+def migrate_data_from_old_model_to_new_model_up(apps, schema_editor):
+    """ Move objects from ContainerDistribution to NewContainerDistribution."""
+    AnsibleDistribution = apps.get_model('container', 'ContainerDistribution')
+    NewAnsibleDistribution = apps.get_model('container', 'NewContainerDistribution')
+    for container_distribution in AnsibleDistribution.objects.all():
+        with transaction.atomic():
+            NewAnsibleDistribution(
+                pulp_id=container_distribution.pulp_id,
+                pulp_created=container_distribution.pulp_created,
+                pulp_last_updated=container_distribution.pulp_last_updated,
+                pulp_type=container_distribution.pulp_type,
+                name=container_distribution.name,
+                base_path=container_distribution.base_path,
+                content_guard=container_distribution.content_guard,
+                remote=container_distribution.remote,
+                repository_version=container_distribution.repository_version
```

Comment:
there are 3 additional fields on the Container distribution which also need data migration `description, namespace, private`